universal-login-commons/lib/core/utils/isValidAmount.ts

```diff
@@ -1,9 +1,11 @@
 import {utils} from 'ethers';
 
-export const isValidAmount = (amount: string, balance: string): boolean => {
-  if (amount.match(/(^[0-9]+(\.?[0-9])*$)/)) {
-    const amountAsBigNumber = utils.bigNumberify(utils.parseEther(amount));
-    return amountAsBigNumber.gt(0) && amountAsBigNumber.lt(utils.parseEther(balance));
+export const isValidAmount = (balance: string | null, amount?: string): boolean => {
```

Comment:
In general it is a bad practice to pass complex type like `string | false`;
It is better to return nullable:
```ts
type Nullable<T> = T | null; 

const resolveName = async (...): Nullable<string> => {
...
}

```

Reply:
Done